### ISSUE TITLE
AS7 projects

### ISSUE THREAD

**Issue by Graynomad** [aethaniel/ExperimentalCore-sam#42]
Can we have a couple of example AS7 projects, maybe blinky and a serial echoing app or something

**Comment by aethaniel** [comment]
Hi @Graynomad, I could add AS7 makefile based projects for the existing examples present in libraries/tests only.
Is that OK for you?

**Comment by Graynomad** [comment]
Yeah I assume so. TBH I'm still trying to get my head around the 10,000 files and examples that ship with AS, so I don't really know what I need, but that sounds OK.

**Comment by aethaniel** [comment]
@Graynomad indeed Atmel Studio provides plenty of ASF examples for Atmel boards but this is another subject. The meaning with this repo is to provide something usable in Arduino.cc IDE or other more complex tools provided by the community.

You can check these links:
http://hackaday.com/2015/10/30/code-craft-using-eclipse-for-arduino-development/
http://hackaday.com/2015/10/01/arduino-development-theres-a-makefile-for-that/
http://www.baeyens.it/eclipse/
